addons/web_editor/static/src/js/editor/odoo-editor/src/commands/commands.js

```diff
@@ -304,7 +314,7 @@ export const editorCommands = {
         selection.removeAllRanges();
         const newRange = new Range();
         let lastPosition = rightPos(currentNode);
-        if (lastPosition[0] === editor.editable) {
+        if (lastPosition[0] === editor.editable || ['UL', 'OL'].includes(lastPosition[0].nodeName)) {
```

Comment:
I don't understand this change because I don't see how it related to the comment line below.